lib/inputs/index.js

```diff
@@ -22,11 +23,13 @@ module.exports = class Inputs {
       this._types.push(new AdzerkPingbacks(records));
       this._types.push(new DovetailDownloads());
       this._types.push(new DovetailImpressions());
+      this._types.push(new RedisIncrements(records));
```

Comment:
I decided to throw the redis stuff into the pingbacks lambda, not the bigquery-inserts, as they're sort of ephemeral and don't need to be retried if they fail.  (Everything in the bigquery lambda gets retried until it succeeds).